contrib/plasma-workspace/files/metadata.json

```diff
@@ -0,0 +1,13 @@
+{
+    "KPlugin": {
+        "Authors": [
+            {
+                "Email": "triallax@tutanota.com",
+                "Name": "triallax"
```

Comment:
i should probably put the author of the wallpapers here instead?

Reply:
put generic info there, i.e. `info@chimera-linux.org` and `Chimera Linux`